### PR TITLE
chore: add frax incentivizer

### PR DESCRIPTION
**Pull request by gosuto-inzasheru**

### CODE REVIEW

File: extras/fraxtal.json [hunk 1]
```diff
@@ -3,6 +3,9 @@
         "zero": "0x0000000000000000000000000000000000000000"
     },
     "tokens": {},
+    "frax": {
+        "incentivizer_amo_v3_2": "0xe9b3f9cd8851216fBb828da590CCA8CE7F03E863"
```
Comment (Tritium-VLK) [inline extras/fraxtal.json:7]: Note that per Nader AMOs are per coin.  So this is the SFRAX AMO, there will likely be other AMOs for other frax tokens. 
```suggestion
        "sfrax_amo_v3_2": "0xe9b3f9cd8851216fBb828da590CCA8CE7F03E863"
```

**Review by Tritium-VLK — CHANGES_REQUESTED** [review]

**Comment by Tritium-VLK** [comment]
image

File: extras/fraxtal.json (hunk 1 above)
Comment (gosuto-inzasheru) [reply extras/fraxtal.json:7]: ah you are right. well half. it is for fxs actually:

Screenshot 2024-09-03 at 13 30 01

Comment (Tritium-VLK) [reply extras/fraxtal.json]: I think all AMOs distribute FXS (and maybe other tokens) but are deployed per base pair token.  I can add you to the chat with Nader if you want to ask more about it.

**Review by Tritium-VLK — APPROVED** [review]

**Pull request merged**